src/core/a-node.js

```diff
@@ -1,5 +1,6 @@
 /* global customElements, CustomEvent, HTMLElement, MutationObserver */
 var utils = require('../utils/');
+var ready = require('./ready');
```

Comment:
`ready` is not a very descriptive word. Can we think of something better?

Reply:
Perhaps `readyState` analogous to the `document.readyState`?

Reply:
aframeReadyState?

Reply:
As the local var name that works, though as file name I'd say `readyState.js` suffices as the aframe prefix won't make things clearer IMHO.